node/silkworm/downloader/internals/header_retrieval.cpp

```diff
@@ -131,27 +132,27 @@ std::tuple<Hash, BlockNum> HeaderRetrieval::get_ancestor(Hash hash, BlockNum blo
     if (ancestorDelta > blockNum) return {Hash{}, 0};
 
     if (ancestorDelta == 1) {
-        auto header = db_tx_.read_header(blockNum, hash);
+        auto header = db::read_header(db_tx_, blockNum, hash);
         if (header)
             return {header->parent_hash, blockNum - 1};
         else
             return {Hash{}, 0};
```

Comment:
Google style : braces can be omitted _if an if statement has no else or else if clauses, then the curly braces for the controlled statement or the line breaks inside the curly braces may be omitted if as a result the entire if statement appears on either a single line (in which case there is a space between the closing parenthesis and the controlled [...]_
https://google.github.io/styleguide/cppguide.html#Conditionals